romeo/src/system.rs

```diff
@@ -371,14 +382,20 @@ async fn burn_asset(
 
 	let tx = StacksTransaction::new(tx_version, tx_auth, tx_payload);
 
-	let txid = stacks_client
-		.lock()
-		.await
-		.sign_and_broadcast(tx)
-		.await
-		.expect("Unable to sign and broadcast the mint transaction");
-
-	Event::BurnBroadcasted(withdrawal_info, txid)
+	match stacks_client.lock().await.sign_and_broadcast(tx).await {
+		Ok(txid) => Event::BurnBroadcasted(withdrawal_info, txid),
+		Err(err) => {
+			if config.strict {
+				panic!(
+					"Unable to sign and broadcast the burn transaction: {}",
+					err
+				);
+			} else {
+				debug!("Ignoring failure to sign and broadcast the burn transaction: {}", err);
+				Event::BurnBroadcasted(withdrawal_info, DUMMY_STACKS_ID)
```

Comment:
It informs the user that the request failed.

Reply:
Ideally, we have a better error handling, in particular events for that so that we can analysis the error and decide on it. Probably, won't happen in the developer release.